src/components/RefinementList/RefinementList.js

```diff
@@ -146,65 +143,103 @@ export class RawRefinementList extends Component {
 
   render() {
     // Adding `-lvl0` classes
-    const cssClassList = [this.props.cssClasses.list];
-    if (this.props.cssClasses.depth) {
-      cssClassList.push(`${this.props.cssClasses.depth}${this.props.depth}`);
-    }
+    const cssClassList = cx(this.props.cssClasses.list, {
+      [`${this.props.cssClasses.depth}${this.props.depth}`]: this.props
+        .cssClasses.depth,
+    });
+
+    const showMoreButtonClassName =
+      this.props.showMore === true && this.props.canToggleShowMore
+        ? this.props.cssClasses.showMore
+        : cx(
+            this.props.cssClasses.showMore,
+            this.props.cssClasses.disabledShowMore
+          );
 
-    const showMoreBtn =
-      this.props.showMore === true && this.props.canToggleShowMore ? (
```

Comment:
Why do we drop this condition `this.props.canToggleShowMore` ?

Reply:
Fixed!

Reply:
It's actually handled by the classname above, right?

From what I understand, if `this.props.canToggleShowMore` is false, we want to add the classname `disabledShowMore`.

Reply:
With the v2 when you can't toggle the button is removed. React & Vue display the button with the disabled state. The first implementation was fine then.